hw/dv/data/common_sim_cfg.hjson

```diff
@@ -23,12 +23,17 @@
   sw_root_dir:        "{proj_root}/sw"
 
   // pass and fail patterns
-  pass_patterns:    ["^TEST PASSED (UVM_)?CHECKS$"]
-  fail_patterns:    ["^UVM_ERROR\\s[^:].*$",
-                     "^UVM_FATAL\\s[^:].*$",
-                     "^Assert failed: ",
-                     "^\\s*Offending '.*'",
-                     "^TEST FAILED (UVM_)?CHECKS$"]
+  compile_pass_patterns: []
+  compile_fail_patterns: ["^Error-.*$",
```

Comment:
Split this for VCS and Xcelium, since those patterns are specific to those simulators. 
In this file:
```
    build_fail_patterns: []
```

In `vcs.hjson`:
```
    build_fail_patterns: ["Error-.*$"]
```

In `xcelium.hjson`:
```
    build_fail_patterns: ["\\*E.*$"]
```

Reply:
Done